shared/nav.js.flow

```diff
@@ -17,6 +17,7 @@ export type Props = {
   routeSelected: Tab,
   routeStack: RouteRenderStack,
   hideNav: boolean,
+  reachability: ?Reachability,
```

Comment:
if i did this i'd have to change the config code and reducer to enforce this. this is just me moving this from global errors